File: bank_system.cpp
#include<iostream>
using namespace std;

class Account
{
    private:
        long AccountNumber;
        string firstname;
        string lastname;
        float balance;
        static long accountnum;
    
    public:
        Account(string fname, string lname, float bal);
        void setAccountNumber();
        void setFirstName(string fname);
        void setLastName(string lname);
        void setBalance(float bal);
        void display();
};

long Account :: accountnum=0;

Account :: Account(string fname, string lname, float bal)
{
    setAccountNumber();
    setFirstName(fname);
    setLastName(lname);
    setBalance(bal);
}

void Account :: setAccountNumber()
{
        accountnum++;
        AccountNumber = accountnum;
}

void Account :: setFirstName(string fname)
{
    firstname = fname;
}

void Account :: setLastName(string lname)
{
    lastname = lname;
}

void Account :: setBalance(float bal)
{
    balance = bal;
}

int main()
{
    Account *acc;
    int n;
    string fname, lname;
    float bal;

    cout << "Enter the total account to create: ";
    cin >> n;

    for(int i=0; i<n; i++)
    {
        cout << "Enter the " << i+1 << "Account To Create " << endl;
        cout << "Enter First Name : ";
        cin >> fname;
        cout << "Enter Last Name : ";
        cin >> lname;
        cout << "Enter Initial Balance : ";
        cin >> bal;

        acc = new Account(fname, lname, bal);
    }

    return 0;
}

File: bank_system.py
DB_HOST = "localhost"
DB_NAME = "mydb"
DB_USER = "azad"
DB_PASS = "azad@#123"

import psycopg2

conn = psycopg2.connect(dbname=DB_NAME, user=DB_USER, password=DB_PASS, host=DB_HOST)
cur = conn.cursor()

def open_account():
    print("Enter First Name : ", end='')
    f_name = input()
    
    print("Enter Last Name : ", end='')
    l_name = input()
    
    print("Enter the Minimum Balance : ", end='')
    bal = int(input())
    
    cur.execute("INSERT INTO bank (firstname, lastname, balance) VALUES(%s, %s, %s);", (f_name, l_name, bal))
    
    

def balance_enquiry():
    cur.execute("SELECT * FROM bank;")
    print(cur.fetchall())

def option_chooser(option):
    switcher={
        1: open_account(),
        2: balance_enquiry(),
        7: 0,
    }
    return switcher.get(option, "Option Not Available")
    
    
cur.execute("CREATE TABLE bank (account serial PRIMARY KEY, firstname varchar(20), lastname varchar(20), balance int);")


while(1):
    print("**************Bank System**************")
    print("\t\t1. Create an Account")
    print("\t\t2. Balance Enquiry")
    print("\t\t3. Deposit")
    print("\t\t4. Withdraw")
    print("\t\t5. Close an Account")
    print("\t\t6. Show all Accounts")
    print("\t\t7. Quit")
    option = int(input())
    
    if(option == 7):
        print("Thanks !!!!!!!!")
        break
    result = option_chooser(option)        

conn.commit()
cur.close()
conn.close()